src/docs/product/sentry-basics/search/searchable-properties/issues.mdx

```diff
@@ -26,13 +26,13 @@ Indicates if the app is in the foreground or background. Values are `1/0` or `tr
 
 ### `assigned`
 
-Returns issues assigned to the defined user. Values can be a user ID (your email address), `me` for yourself, `me_or_none` for yourself or no assignee, or # followed by the team name, that is `#team-name`.
+Returns issues assigned to the defined user(s) or team(s). Values can be a user ID (your email address), `me` for yourself, `none` for no assignee, `my_teams` or `#team-name` for teams you belong to.
 
 - **Type:** team or org user
 
 ### `assigned_or_suggested`
 
-Returns issues assigned to or suggested to be assigned to the defined user. Suggested assignees are determined by matching [ownership rules](/product/issues/ownership-rules/) and [suspect commits](/product/issues/suspect-commits/). Values can be a user ID (your email address), `me` for yourself, `me_or_none` for yourself or no assignee/suggestions, or # followed by the team name, that is `#team-name`.
+Returns issues that are either assigned to or suggested as something that should be assigned to defined user(s) or team(s). Suggested assignees are found by matching [ownership rules](/product/issues/ownership-rules/) with [suspect commits](/product/issues/suspect-commits/). Values can be a user ID (your email address), `me` for yourself, `none` for no assignee/suggestion, `my_teams` for `#team-name` for teams you belong to.
```

Comment:
nit: maybe 'Returns issues that are assigned to or suggest to be assigned to defined user(s) or teams(s). '